test/ILLink.RoslynAnalyzer.Tests/TestCaseUtils.cs

```diff
@@ -209,5 +260,87 @@ internal static void GetDirectoryPaths (out string rootSourceDirectory, out stri
 			rootSourceDirectory = Path.GetFullPath (Path.Combine (artifactsBinDir, "..", "..", "test", "Mono.Linker.Tests.Cases"));
 			testAssemblyPath = Path.GetFullPath (Path.Combine (artifactsBinDir, "ILLink.RoslynAnalyzer.Tests", configDirectoryName, tfm));
 		}
+
+		internal static Task VerifyCodeFix<TAnalyzer, TCodeFix> (
+			string source,
+			string fixedSource,
+			DiagnosticResult[] baselineExpected,
+			DiagnosticResult[] fixedExpected,
+			int? numberOfIterations = null)
+			where TAnalyzer : DiagnosticAnalyzer, new()
+			where TCodeFix : CodeFixProvider, new()
+		{
+			string analyzerAttribute = GetAttributeDefinition<TAnalyzer> ();
+			string codeFixAttribute = GetAttributeDefinition<TCodeFix> ();
+			var attributeDefinitions = analyzerAttribute;
+			if (!string.Equals (analyzerAttribute, codeFixAttribute)) {
+				attributeDefinitions += codeFixAttribute;
```

Comment:
Do we need this to be conditional?

Reply:
Yes, adding two times the same definition will make the compiler to error on ambiguity